packages/protocol/contracts/common/Proxy.sol

```diff
@@ -138,4 +139,20 @@ contract Proxy {
     }
     emit OwnerSet(newOwner);
   }
+
+
+  /**
+   * @dev isContract detect whether the address is 
+   *      a contract address or externally owned account (EOA)
+   * WARNING: Calling this function from a constructor will return 0
+  *           independently if the address given as parameter is a contract or EOA
+   * @return true if it is a contract address
+   */
+  function isContract(address addr) public view returns (bool) {
+    uint size;
```

Comment:
Nit: by convention, we use `uint256` throughout the code base.

Reply:
:+1:

Reply:
:+1: